docs/metrics/scheduler.mdx

```diff
@@ -0,0 +1,327 @@
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+## Container
+### Total allocation attempt
+Total number of attempts to allocate containers.
+State of the attempt includes `allocated`, `rejected`, `error`, `released`.
+
+**Metric Type**: `counter`
+
+**Namespace**: `yunikorn`
+
+**Subsystem**: `scheduler`
+
+```json
+yunikorn_scheduler_container_allocation_attempt_total{state="allocated"} 0
+yunikorn_scheduler_container_allocation_attempt_total{state="error"} 0
+yunikorn_scheduler_container_allocation_attempt_total{state="released"} 0
+```
+
+## Application
+### Total
+Total number of applications.
+State of the application includes `running`, `failed` and `completed`.
+
+**Metric Type**: `gauge`
+
+**Namespace**: `yunikorn`
+
+**Subsystem**: `scheduler`
+```
+yunikorn_scheduler_application_total{state="running"} 0
+```
+
+### Total Submission
+Total number of application submissions.
+State of the attempt includes `accepted` and `rejected`.
+
+**Metric Type**: `counter`
+
+**Namespace**: `yunikorn`
+
+**Subsystem**: `scheduler`
+```
+yunikorn_scheduler_application_submission_total{result="accepted"} 6
+```
+
+## Latency
+### Scheduling latency
+Latency of the main scheduling routine, in milliseconds.
+This metric includes latencies, such as `Node sorting`, `Trynode` and `Trypreemption`. 
+
+**Metric Type**: `histogram`
+
+**Interval**: `millisecond`
+
+**Namespace**: `yunikorn`
+
+**Subsystem**: `scheduler`
+
+```
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="0.0001"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="0.001"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="0.01"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="0.1"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="1"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="10"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_bucket{le="+Inf"} 0
+yunikorn_scheduler_scheduling_latency_milliseconds_sum 0
+yunikorn_scheduler_scheduling_latency_milliseconds_count 0
+```
+
+### Node sorting
+Latencies including `node sorting`, `application sorting` and `queue sorting`, in milliseconds.
+
+**Metric Type**: `histogram`
+
+**Interval**: `millisecond`
+
+**Namespace**: `yunikorn`
+
+**Subsystem**: `scheduler`
+
+```mdx-code-block
+<Tabs
+  defaultValue="node_sorting"
+  values={[
+    { label: 'Node sorting', value: 'node_sorting'},
+    { label: 'App sorting', value: 'app_sorting'},
+    { label: 'Queue sorting', value: 'queue_sorting'},
+  ]}>
+<TabItem value="app_sorting">
+
+```json
+/*
```

Comment:
Want to know if this comment symbol is necessary? 
If there is no comment symbol it will allow the markdown to be colored automatically, just like the other examples.